azure-pipelines.yml

```diff
@@ -0,0 +1,77 @@
+# Node.js
+# Build a general Node.js project with npm.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+jobs:
+  - job: Linux_Node8
+
+    pool:
+      vmImage: "ubuntu 16.04"
+
+    steps:
+      - task: NodeTool@0
+        inputs:
+          versionSpec: "8.x"
+        displayName: "Install Node.js"
+
+      - script: |
+          npm ci
+          npm run build
+        displayName: "npm install and build"
+
+      - script: |
+          npm test
+        displayName: "npm test"
+
+      - script: |
+          npm run lint
+        displayName: "npm run lint"
+
+      - script: |
+          ENGINE_API_KEY=$CHECKS_API_KEY ./packages/apollo/bin/run client:check
+        displayName: "Query Check"
+
+  - job: Linux_Node10
+
+    pool:
+      vmImage: "ubuntu 16.04"
+
+    steps:
+      - task: NodeTool@0
+        inputs:
+          versionSpec: "10.x"
+        displayName: "Install Node.js"
+
+      - script: |
+          npm ci
```

Comment:
Should we make an npm script similar to what we do for `npm run circle`?

`"azure": "npm ci && npm run build && npm test"`

This would clean up some duplication and give devs an easy way to run exactly the commands the pipeline does without having to look anywhere besides package.json